compiler/rustc_codegen_rmc/src/compiler_interface.rs

```diff
@@ -69,12 +69,14 @@ impl CodegenBackend for GotocCodegenBackend {
                         c.call_with_panic_debug_info(
                             |ctx| ctx.declare_function(instance),
                             format!("declare_function: {}", c.readable_instance_name(instance)),
+                            Some(c.codegen_span(&tcx.instance_mir(instance.def).span)),
                         );
                     }
                     MonoItem::Static(def_id) => {
                         c.call_with_panic_debug_info(
                             |ctx| ctx.declare_static(def_id, item),
                             format!("declare_static: {:?}", def_id),
+                            None,
```

Comment:
IS there no location for static variables?

Reply:
In `codegen/static_vars.rs` we do 
```rust
self.tcx.def_span(def_id);
```

It might actually just work to do this for all the cases where we need locations?